gufe/mapping/ligandatommapping.py

```diff
@@ -76,11 +76,10 @@ def annotations(self):
     def _to_dict(self):
         """Serialize to dict"""
         return {
-            # openff serialization doesn't go deep, so stringify at this level
-            'componentA': self.componentA.to_dict(),
-            'componentB': self.componentB.to_dict(),
+            'componentA': self.componentA,
+            'componentB': self.componentB,
             'componentA_to_componentB': self._compA_to_compB,
-            'annotations': json.dumps(self.annotations),
+            'annotations': json.dumps(self._annotations, sort_keys=True),
```

Comment:
do we want to pass this through `cls=JSON_HANDLER.encoder`? (and similarly on the return) If an annotation had units, this would currently fail; adding the `cls` arg to dumps/loads is all that is needed to fix that.